monai/networks/utils.py

```diff
@@ -516,6 +516,8 @@ def copy_model_state(
     unchanged_keys = sorted(set(all_keys).difference(updated_keys))
     logger.info(f"'dst' model updated: {len(updated_keys)} of {len(dst_dict)} variables.")
     if inplace and isinstance(dst, torch.nn.Module):
+        if isinstance(dst, (nn.DataParallel, nn.parallel.DistributedDataParallel)):
```

Comment:
I think line 496 already covered this logic:
```py
dst_dict = OrderedDict(get_state_dict(dst))
```

Thanks.

Reply:
This is to modify dst instead of dst_dict, not sure how 496 is useful here...

Reply:
Oh, I misunderstood the logic..

Thanks.